tests/search/search.spec.js

```diff
@@ -0,0 +1,73 @@
+import { test, expect } from "@playwright/test";
+
+const homePageUrl = "https://magento.softwaretestingboard.com/";
+const searchBar = "Search entire store here...";
+const elementsList = "ul[role='listbox']";
+const searchItems = ".qs-option-name";
+const amount = ".amount";
+const products = ".product-item";
+const pagination = ".items.pages-items";
+
+test("Search in header for pants", async ({ page }) => {
+  await page.goto(homePageUrl);
+
+  // Enter "pants" and get sugestions
+  await page
+    .getByPlaceholder(searchBar)
+    .pressSequentially("pants", { delay: 300 });
+  await expect(page.locator(elementsList)).toBeVisible();
+  const sugestionList = await page.locator(searchItems).all();
+
+  // Get sugestions
+  let sugestions = [];
+  for (let sugestion of sugestionList) {
+    sugestions.push(await sugestion.innerText());
+  }
```

Comment:
Is this necessary? all already returns a list of array. Can you make the assertion on that list?

Reply:
nevermind adi already wrote the code